eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovCertSpec.hs

```diff
@@ -107,6 +107,41 @@ spec = do
       -- Confirm that after registering a hot key, they are active
       _hotKey <- registerCommitteeHotKey cc
       ccShouldNotBeResigned cc
+  describe "Removing CC with UpdateCommittee" $ do
+    it "Non registered" $ do
+      (drepCred, _, _) <- setupSingleDRep 1_000_000
+      (spoC, _, _) <- setupPoolWithStake $ Coin 42_000_000
+      passNEpochs 2
+      initialCommittee <- getCommitteeMembers
+      logToExpr initialCommittee
+      initialCommittee `shouldSatisfy` not . Set.null
+      removeCCGaid <- submitUpdateCommittee Nothing initialCommittee mempty (1 %! 2)
+      submitYesVote_ (DRepVoter drepCred) removeCCGaid
+      submitYesVote_ (StakePoolVoter spoC) removeCCGaid
+      passNEpochs 2
+      finalCommittee <- getCommitteeMembers
+      logToExpr finalCommittee
+      finalCommittee `shouldSatisfy` Set.null
+    it "Registered" $ do
+      (drepCred, _, _) <- setupSingleDRep 1_000_000
+      (spoC, _, _) <- setupPoolWithStake $ Coin 42_000_000
+      passNEpochs 2
+      initialCommittee <- getCommitteeMembers
+      logToExpr initialCommittee
+      initialCommittee `shouldSatisfy` not . Set.null
+      forM_ (Set.toList initialCommittee) $ \kh -> do
+        ccHotCred <- KeyHashObj <$> freshKeyHash
+        submitTx_ $
+          mkBasicTx mkBasicTxBody
+            & bodyTxL . certsTxBodyL
+              .~ SSeq.singleton (AuthCommitteeHotKeyTxCert kh ccHotCred)
+      removeCCGaid <- submitUpdateCommittee Nothing initialCommittee mempty (1 %! 2)
```

Comment:
A more intereseting test case would be to remove a subset of a committee and add a new committee member.
```suggestion
      newCommittee <- arbitrary
      initialSubCommittee <- arbitrarySubSet initialCommittee
      removeCCGaid <- submitUpdateCommittee newCommittee initialSubCommittee mempty (1 %! 2)
```

Then check in the end the committee was updated accordingly